app/src/main/cpp/skyline/vfs/ticket.h

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2021 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#pragma once
+
+#include <vfs/backing.h>
+#include <crypto/key_store.h>
+
+namespace skyline::vfs {
+    struct Ticket {
+        enum class TitleKeyType : u8 {
+            Common = 0x0,
+            Personal = 0x1
```

Comment:
You should have per-member comments explaining the differences here:
```suggestion
            Common = 0x0,   //!< The title key will be stored as a 16-byte block
            Personal = 0x1, //!< The title key will be stored as a personalized RSA-2048 message
```